packages/web/index.js

```diff
@@ -21,6 +21,11 @@ module.exports = (neutrino, opts = {}) => {
     env: false,
     hot: true,
     html: {},
+    devtool: {
+      development: 'cheap-module-eval-source-map',
+      production: false,
+      test: 'source-map'
```

Comment:
This changes the `NODE_ENV=test` devtool from the previous value of `undefined` (which would have given a value of `eval` in `mode: 'development'` due to the webpack defaulter), to `'source-map'`. Have you done comparisons on the quality vs speed trade-offs of the two? (for both one-off test runs and watching; guessing it needs to be for karma since jest/mocha circumvent webpack) Since we don't minify for tests, I suppose there is slightly less benefit of 'source-map' over the cheaper options.

Reply:
I haven't done any benchmarking, I just added this to make testing easier to debug.

Reply:
Trying with Treeherder, the webpack compilation part of `yarn test` (as judged by the duration output to the console) varied accordingly (lowest of several repeats listed):
* `devtool: false`: 9060ms
* `devtool: 'cheap-module-eval-source-map'`: 9641ms
* `devtool: 'source-map'`: 11265ms

'source-map' means a 25% increase in compile time, but still not that significant proportion of the total `yarn test` time (which was ~20s), given the time taken to spin up the browser and run the tests.

As such, `source-map` seems fine to me :-)